Адаптивная сетка для главной страницы

Valeriia-2 opened the pull request:
---
:mortar_board: [Адаптивная сетка для главной страницы](https://up.htmlacademy.ru/adaptive/25/user/1445103/tasks/7)

:boom: https://htmlacademy-adaptive.github.io/1445103-pink-25/5/

Valeriia-2 committed:
Ограничение ширины блока advantages

Valeriia-2 committed:
Ограничила ширину блока reviews

Valeriia-2 committed:
Ограничила ширину блока contacts

Valeriia-2 committed:
Подправила сетку футера

Valeriia-2 committed:
Поправила сетку блока efficiency

Valeriia-2 committed:
Поправки для блока reviews

Valeriia-2 committed:
Поправки страницы форм

feralist21 commented on source/less/blocks/container.less:
```diff
@@ -3,7 +3,7 @@
 
   @media (min-width: @tablet-width) {
-    max-width: 660px;
+    max-width: 640px;
```
Исходя из планшетной версии макета ширина контейнера в данному случае должна быть 620px. Планшетная ширина 660px - 40px на горизонтальные падинги = 620px.

feralist21 reviewed (changes requested):
Внести пожалуйста небольшие правки по контейнеру.

Valeriia-2 committed:
Изменила ширину контейнеров, добавила паддинги

Valeriia-2 committed:
Изменила ширины переменных

Valeriia-2 committed:
Поменяла ширину .main-header__content-wrapper

Valeriia-2 committed:
Зафиксировала ширину у page-main__titles

Valeriia-2 committed:
Убрала паддинг у advantages__list

Valeriia-2 committed:
Ограничила ширину для таблицы

П.С В этом случае таблица на мобильной версии больше не тянется. Но я делаю фикс

Valeriia-2 committed:
Контейнер для page-main__subtitle-wrapper

Valeriia-2 committed:
Страница каталога - поправки контейнер

Valeriia-2 committed:
Страница каталога- убрала зазорчик

feralist21 reviewed (approved):
Все хорошо.
